src/server/seeds/development/05_orders.js

```diff
@@ -0,0 +1,28 @@
+exports.seed = function (knex) {
+  return knex('orders')
+    .del()
+    .then(function () {
+      return knex('orders').insert([
+        {
+          user_id: 1,
+          created_at: '2021-09-12 15:00:00',
+        },
```

Comment:
Bikram, **staus** is  notNullable. So Orders need **status** in seeding.

Reply:
Hi @zmzade i left that value as default value, thanks for review 👍

Reply:
ok, correct.